upgrade v14: add instructions for updating vocabularies in v14 (#923)
inveniosoftware/docs-invenio-rdm: docs/releases/vNext/upgrade-vNext.md

         - `cluster:admin/component_template/put`
         - `indices:admin/index_template/put`

### Update vocabularies

> A number of vocabularies have been enhanced with terms from the DataCite 4.4-4.7 releases, as well as other mapping improvements and translations. In order to update the vocabularies in your repository, you'll need to reload the fixtures with a command such as `invenio rdm-records add-to-fixture contributorsroles`. Vocabularies that have updates include:

Reviewer: Just being potentially even more clear and explicit.

```suggestion
A number of out-of-the-box vocabularies have been enhanced with terms from the DataCite 4.4-4.7 releases, as well as other mapping improvements and translations. In order to update these in your repository, you'll need to reload their fixtures with `invenio rdm-records add-to-fixture <vocabulary fixture>`. The `<vocabulary fixture>` for vocabularies that have updates are:
```

- datetypes
- descriptiontypes
- licenses
- relationtypes
- resourcetypes
- contributorsroles
- creatorsroles
- titletypes

> If you've customized any of these vocabularies for your instance, you'll need to merge changes from the source files in invenio-rdm-records into the custom vocabulary files in your instance before running the `add-to-fixture` command.

Reviewer: ```suggestion
If you've customized any of these vocabularies for your instance, you'll need to merge changes from the [source files in invenio-rdm-records](https://github.com/inveniosoftware/invenio-rdm-records/tree/master/invenio_rdm_records/fixtures/data/vocabularies) into the custom vocabulary files in your instance before running the `add-to-fixture` command.
```

(that link is pointing to master, but maybe should point to v14 branch when it's out, it's probably fine anyway)
